src/wallet/rpcwallet.h

```diff
@@ -31,4 +31,5 @@ bool EnsureWalletIsAvailable(CWallet *, bool avoidException);
 UniValue getaddressinfo(const JSONRPCRequest& request);
 UniValue signrawtransactionwithwallet(const JSONRPCRequest& request);
 bool FillPSBT(const CWallet* pwallet, PartiallySignedTransaction& psbtx, const CTransaction* txConst, int sighash_type = 1, bool sign = true, bool bip32derivs = false);
+bool ParseHDKeypath(std::string keypath_str, std::vector<uint32_t>& keypath);
```

Comment:
Rather than declare it in the header, you could declare it `extern` in the test file.